res/css/views/elements/_Dropdown.scss

```diff
@@ -105,24 +108,19 @@ input.mx_Dropdown_option:focus {
     position: absolute;
     left: -1px;
     right: -1px;
-    top: 100%;
+    top: calc(100% + 2.5px);
     z-index: 2;
     margin: 0;
     padding: 0px;
-    border-radius: 4px;
-    border: 1px solid $accent-alt;
+    border-radius: 8px;
+    border: 1px solid $quinary-content;
     background-color: $background;
     max-height: 200px;
     overflow-y: auto;
 }
 
-.mx_Dropdown_menu .mx_Dropdown_option {
-    height: auto;
-    min-height: 35px;
-}
-
 .mx_Dropdown_menu .mx_Dropdown_option_highlight {
-    background-color: $focus-bg-color;
+    background-color: rgba($quaternary-content, 0.2);
```

Comment:
This won't work for custom themes where it'll base itself on the inherited parent theme instead, rgba runs in postcss at build time